src/pages/plugins/concepts/manifest/index.md

```diff
@@ -0,0 +1,863 @@
+<code class="language-json">{`{
+    "domains": "all"
+}
+`}</code>
+<h4>Properties</h4>
+<table>
+    <thead>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Required</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><inlineCode>domains</inlineCode> *</td>
+        <td><inlineCode>string[] | "all"</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The domains that the plugin can access. Can be a list of domains, or the string "all" to allow access to all domains.</p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### WebviewPermission
+
+<p>Enables the plugin to use webviews in its UI to display web content or complex UI.</p>
+<p><strong>Example</strong></p>
+<code class="language-json">{`{
+    "allow": "yes",
+    "domains": ["https://example.com"],
+    "enableMessageBridge": "localAndRemote"
+}
+`}</code>
+<p>Then, in your plugin code, you can use a webview like this:</p>
+<code class="language-html">{`<webview src="https://example.com" />
+`}</code>
+<p>To communicate between the webview and the plugin, you can use the message API:</p>
+<code class="language-javascript">{`// In the plugin:
+const webview = document.querySelector("webview");
+webview.addEventListener("message", (event) => \{
+    console.log("Received message from webview:", event.data);
+    webview.postMessage("Hello from the plugin!");
+\});\n
+// In the webview:
+window.addEventListener("message", (event) => \{
+    console.log("Received message from plugin:", event.data);
+    window.uxpHost.postMessage("Hello from the webview!");
+\});
+`}</code>
+<h4>Properties</h4>
+<table>
+    <thead>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Required</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><inlineCode>allow</inlineCode> *</td>
+        <td><inlineCode>"yes"</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>Must be set to "yes" to enable webviews.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>domains</inlineCode> *</td>
+        <td><inlineCode>string[] | "all"</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The domains that the plugin can access. Can be a list of domains, or the string "all" to allow access to all domains.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>enableMessageBridge</inlineCode></td>
+        <td><inlineCode>"no" | "localAndRemote"</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Specifies whether the plugin can communicate with the webview using the message API.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>"no"</inlineCode></p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### LaunchProcessPermission
+
+<p>Specifies the schemas and extensions that the plugin can launch.</p>
```

Comment:
Mention recipe: Recipe https://github.com/AdobeDocs/uxp-indesign/blob/6d1ad4ff1c2e096d01edaf91809cd8b4c8f4cad7/src/pages/resources/recipes/launch-process.md

Reply:
Not sure because we should probably keep this document as host-app-unspecific as possible.